server/server/src/lib.rs

```diff
@@ -144,11 +143,7 @@ async fn run_server(
     let service = SettingsService {};
     let service_context = service_provider.context().unwrap();
     let db_settings = service.sync_settings(&service_context).unwrap();
-    let sync_settings = if prefer_config_settings {
```

Comment:
I didn't think that prefer_config_settings was needed, the reason being, if setting was already set in database, this would mean that we already tried to sync with those credentials, and possibly have some data from the data file, overwriting that would be bad